symmetric-core/src/main/java/org/jumpmind/symmetric/util/PropertiesUtil.java

```diff
@@ -41,14 +43,27 @@ public static String getEnginesDir() {
         return enginesDir;
     }
 
-    public static File findPropertiesFileForEngineWithName(String engineName) {
+    public static File findPropertiesFileForEngineWithName(String engineName, Map<String, String> replacementValues) {
+        if (replacementValues == null) {
```

Comment:
Can you add back the old method signature that takes just an engineName?  That preserves the old API for compatibility and makes calling code a little cleaner.